init.lua

```diff
@@ -38,6 +38,13 @@ local table = {
     remove  = table.remove
 }
 local helpers = require("vicious.helpers")
+local dstatus, debug = pcall(require, "gears.debug")
```

Comment:
Shouldn't naughty.notify be used here instead?

Reply:
I can change it, but wouldn't that spam notifications?

Reply:
I was genuinely asking :smiley:  It makes sense to log instead then.